tests/unit_tests.rs

```diff
@@ -1,14 +1,22 @@
 extern crate rsdparsa;
 
+
+#[cfg(test)]
+fn check_parse_and_serialize(sdp_str: &str) {
+    let sdp = rsdparsa::parse_sdp(sdp_str, true);
+    assert!(sdp.is_ok());
+    assert_eq!(sdp.unwrap().to_string(), sdp_str.to_string())
+}
+
 #[test]
 fn parse_minimal_sdp() {
-    let sdp = "v=0\r\n
-o=- 0 0 IN IP4 0.0.0.0\r\n
-s=-\r\n
-c=IN IP4 0.0.0.0\r\n
-t=0 0\r\n
-m=audio 0 UDP/TLS/RTP/SAVPF 0\r\n";
-    let sdp_res = rsdparsa::parse_sdp(sdp, true);
+    let sdp_str =  "v=0\r\n\
+                    o=- 0 0 IN IP4 0.0.0.0\r\n\
```

Comment:
Nice formatting, but I think you introduced now lots of leading white spaces. The rsdparsa will happily ignore them. But officially that is not legal SDP. So I don't think we should test here with invalid SDP (except in the one test case which on purpose adds white spaces all over the place).

Reply:
This wont create whitespaces, The "\\" at the end of a line skips every whitespace until another non-whitespace appears. Also, the version before always had 2 newlines at the end of every line.